src/tools/searchRepositoryIssuesTool.ts

```diff
@@ -13,7 +12,7 @@ import { generalOrganizationMistakes, generalRepositoryMistakes } from '../rules
 
 const rules = `
   This is the primary tool for investigating general code quality concerns (e.g. best practices, performance, complexity, style) but NOT security issues. 
-  For security-related issues, use the ${searchSecurityItemsTool.name} tool instead. 
+  For security-related issues, use the SRM tools instead. 
```

Comment:
will it know what are the "SRM tools" tho? is SRM a common term we use in descriptions?

Reply:
there's one for organization and one for repositories, and it depends not only on the scope but also on the scan type... so, instead of complicating or restricting, I left it open